diagnostic.md

```diff
@@ -6,70 +6,71 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+resources :books, only: [:index]
+get '/books/' => '#id'
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+For the command above, db is a namespace and migrate is the task being defined.  Essentially, the command executes the rake script with the db:migrate being the context of the bundle
```

Comment:
`bundle exec rake db:migrate` executes all of the database migration files in `db/migrate` and generates a new database schema file, `db/schema.rb`.